plugins/fakeobjects/plugin.js

```diff
@@ -161,7 +161,7 @@
 
 		var realElementHtml = decodeURIComponent( fakeElement.data( 'cke-realelement' ) ),
 			filteredHtml = filterHtml( this, realElementHtml ),
-			realElement = CKEDITOR.dom.element.createFromHtml( filteredHtml, this.document );
+			realElement = CKEDITOR.dom.element.createFromHtml( CKEDITOR.tools.htmlSafeByReview(filteredHtml, 'Data pulled from controlled "fake" element and then filtered.'), this.document );
```

Comment:
What are fake elements and how are they used?

Reply:
Legacy converted